File: OneMonthVibeCoding/CustomerMangementAPI/Models/Order.cs
using System;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace CustomerMangementAPI.Models
{
    public class Order
    {
        [Key]
        public int OrderId { get; set; }
        [ForeignKey("Client")]
        public int ClientId { get; set; }
        public decimal TotalCost { get; set; }
        public string CreateBy { get; set; }
        public DateTime CreateDate { get; set; }
        public DateTime DeliveryDate { get; set; }
        public bool Canceled { get; set; }
        public bool Refund { get; set; }
        public decimal? RefundCost { get; set; }
        public string UpdateBy { get; set; }
        public DateTime? UpdateDate { get; set; }
        public bool IsDeleted { get; set; }
        public virtual Client Client { get; set; }
    }
}

File: OneMonthVibeCoding-UI/customer-management-ui/src/components/orders/domain/Order.ts
import type { Customer } from '../../customers/domain/Customer';

export interface Order {
  orderId: number;
  clientId: number;
  totalCost: number;
  createBy: string;
  createDate: string;
  deliveryDate: string;
  canceled: boolean;
  refund: boolean;
  refundCost: number;
  updateBy: string;
  updateDate: string;
  isDeleted?: boolean;
  client?: Customer;
}
